src/boot/bootctl.c

```diff
@@ -52,6 +52,9 @@ static bool arg_print_dollar_boot_path = false;
 static bool arg_touch_variables = true;
 static PagerFlags arg_pager_flags = 0;
 static bool arg_graceful = false;
+static enum { MID_NO, MID_YES, MID_AUTO, } arg_make_mid = MID_AUTO;
+
+static sd_id128_t machine_id;
```

Comment:
no needless global variables please. sd_id128_get_machine_id() caches internally anyway, another level of caching is really not necessary

Reply:
Oh, I didn't know it cached. Moved to acquisition down-user.